backend/app/ml_routes.py

```
@@ -0,0 +1,194 @@
+
+# Get API key from environment variable
+GOOGLE_API_KEY = os.getenv("GOOGLE_API_KEY", "")
+if not GOOGLE_API_KEY:
+    logger.warning("GOOGLE_API_KEY not found in environment variables")
+
+ml_service = MedicationComparisonService(GOOGLE_API_KEY) if GOOGLE_API_KEY else None
+
+@router.post("/compare-medications")
+async def compare_medications(
+    user_id: str = Form(...),
+    reference_image: UploadFile = File(...),
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)